pkgs/data/fonts/iosevka/default.nix

```diff
@@ -67,7 +66,13 @@ buildNpmPackage rec {
 
   npmDepsHash = "sha256-Ncf07ggyOnz/2SpgdmaYS2X/8Bad+J2sz8Yyx9Iri3E=";
 
-  nativeBuildInputs = [ nodejs remarshal ttfautohint-nox ];
```

Comment:
As the removal of `nodejs` isn't required for fixing Darwin, wonder if it's worth splitting into its own commit...? (cc @wegank, what do you think?)

Reply:
Done.

Reply:
I think splitting into atomic commits is most of the times a very good idea! Thank you @veehaitch for doing it.